client/components/tooltip/test/index.test.tsx

```diff
@@ -82,6 +88,111 @@ describe( 'Tooltip', () => {
 		expect( handleHideMock ).toHaveBeenCalled();
 	} );
 
+	it( 'renders and hides its content when clicked and ignoreMouseHover = true', () => {
+		const handleHideMock = jest.fn();
+		render(
+			<Tooltip
+				content="Tooltip content"
+				onHide={ handleHideMock }
+				ignoreMouseHover
+			>
+				<span>Trigger element</span>
+			</Tooltip>
+		);
+
+		jest.runAllTimers();
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+
+		act( () => {
+			userEvent.click( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect( screen.queryByText( 'Tooltip content' ) ).toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+
+		act( () => {
+			userEvent.click( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+		expect( handleHideMock ).toHaveBeenCalled();
+	} );
+
+	it( 'renders and hides its content when hovered', () => {
+		const handleHideMock = jest.fn();
+		render(
+			<Tooltip content="Tooltip content" onHide={ handleHideMock }>
+				<span>Trigger element</span>
+			</Tooltip>
+		);
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+
+		act( () => {
+			fireEvent.mouseOver( screen.getByText( 'Trigger element' ) );
+			jest.runAllTimers();
+		} );
+
+		expect( screen.queryByText( 'Tooltip content' ) ).toBeInTheDocument();
+		expect( handleHideMock ).not.toHaveBeenCalled();
+
+		act( () => {
+			fireEvent.mouseLeave( screen.getByText( 'Trigger element' ) );
+			jest.advanceTimersByTime( 1000 );
+		} );
+
+		expect(
+			screen.queryByText( 'Tooltip content' )
+		).not.toBeInTheDocument();
+		expect( handleHideMock ).toHaveBeenCalled();
+	} );
+
+	it( 'remains hidden when hovered ignoreMouseHover = true', () => {
```

Comment:
Added a test to ensure NO show/hide on hover events when ignoreMouseHover = true.